### ISSUE TITLE
sonarcloud_user_group_permissions import not implemented

### ISSUE THREAD

**Issue by HorizonNet** [rewe-digital/terraform-provider-sonarcloud#118]
### Terraform Version

```
terraform -v                                                      
Terraform v1.3.6
on darwin_amd64
+ provider registry.terraform.io/hashicorp/azuread v2.31.0
+ provider registry.terraform.io/hashicorp/azurerm v3.34.0
+ provider registry.terraform.io/rewe-digital/sonarcloud v0.5.0
```

### Affected Resource(s)

* sonarcloud_user_group_permissions

### Terraform Configuration Files

```tf
resource "sonarcloud_user_group_permissions" "owners" {
  name        = "Owners"
  permissions = ["admin"]
}
```

```
terraform import sonarcloud_user_group_permissions.owners "Owners"
...
╷
│ Error: Resource Import Not Implemented
│ 
│ This resource does not support import. Please contact the provider developer for additional information.
╵
```

### Expected Behavior

Based on the [documentation](https://registry.terraform.io/providers/rewe-digital/sonarcloud/latest/docs/resources/user_group_permissions) I would have expected that an import of an existing resource is possible.

### Actual Behavior

Import fails.

**Comment by reinoudk** [comment]
Hi @HorizonNet, thanks for reporting this.

I would have expected this to work as well, as the `ImportState` method has been implemented, but I now see that I messed up the return type of resource. The framework needs a `ResourceWithImportState` type instead of just a `Resource` type.

I'll fix this ASAP, sorry for the inconvenience.

**Comment by reinoudk** [comment]
This has been fixed in #119 and was released in `v0.5.1`.
